cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/LedgerEvents/Gov/PredefinedAbstainDRep.hs

```diff
@@ -133,8 +140,25 @@ delegateToAlwaysAbstain
   -> PaymentKeyInfo
   -> StakingKeyPair
   -> m ()
-delegateToAlwaysAbstain execConfig epochStateView configurationFile socketPath sbe work prefix
-                        payingWallet skeyPair@(StakingKeyPair vKeyFile _sKeyFile) = do
+delegateToAlwaysAbstain execConfig epochStateView configurationFile socketPath sbe work prefix =
+  delegateToAutomaticDRep execConfig epochStateView configurationFile socketPath sbe work prefix
+                          "--always-abstain"
+
+delegateToAutomaticDRep
+  :: (MonadTest m, MonadIO m, H.MonadAssertion m, MonadCatch m)
+  => H.ExecConfig
+  -> EpochStateView
+  -> FilePath
+  -> FilePath
+  -> ShelleyBasedEra ConwayEra
+  -> FilePath
+  -> String
```

Comment:
Can you add haddocks to parameters, please?

Reply:
Fixed here: https://github.com/IntersectMBO/cardano-node/pull/5809/commits/a4c063fe379346124dea834b0b10ce6073ab2d48